.cargo-deny-config.toml

```diff
@@ -4,10 +4,6 @@
 # More documentation for the licenses section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
-default = "deny"
-unlicensed = "deny"
-copyleft = "deny"
-allow-osi-fsf-free = "neither"
```

Comment:
shouldn't we keep these?

Reply:
I [updated](https://github.com/smithy-lang/smithy-rs/pull/3866/files#diff-a82124293c890eca3d2616346ac26f7333828853cc5d790b3371b571dffd6a38) `cargo-deny` in the docker image to try and fix the panic happening due to some edge case triggered by `aws-smithy-http-client`. These are all the default now and an `error` on latest version. See https://github.com/EmbarkStudios/cargo-deny/pull/611